python/ray/runtime_env/runtime_env.py

```diff
@@ -349,16 +355,19 @@ def __init__(
         if not _validate:
             return
 
-        if self.get("conda") and self.get("pip"):
+        use_conda_install = int(self.get("conda") is not None)
+        use_pip_install = int(self.get("pip") is not None)
+        use_uv_install = int(self.get("uv") is not None)
+        if use_conda_install + use_pip_install + use_uv_install > 1:
```

Comment:
I think you can directly do

```
(self.get("conda") is not None) + (self.get("pip") is not None) + (self.get("uv") is not None)
```

Since Python supports `True + True + False`

Reply:
Ah nice, good to know!